tests/Xamarin.Forms-Performance-Integration/Droid/Properties/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0" package="Xamarin.Forms_Performance_Integration">
 	<uses-sdk android:minSdkVersion="16" android:targetSdkVersion="30" />
-	<application android:label="Xamarin.Forms.Performance.Integration"></application>
+	<application android:debuggable="true" android:label="Xamarin.Forms.Performance.Integration"></application>
```

Comment:
Does this project run instrumentation tests? Does it need `debuggable` for `run-as`?

Reply:
we need `debuggable` for `run-as` to work.

Reply:
This project doesn't have any tests? (instrumentation) Is `run-as` used for something else?

Reply:
The `run-as` is used to pull the Test results out of the app cache directory on the device.

Reply:
or the instrumentation output.

Reply:
I don't think this project has any test results:

https://github.com/xamarin/xamarin-android/blob/2e9245299f7d28a1740f87b7eb66529cf5917499/tests/Xamarin.Forms-Performance-Integration/Droid/Xamarin.Forms.Performance.Integration.Droid.projitems#L10

Several of the others do:

https://github.com/xamarin/xamarin-android/blob/2e9245299f7d28a1740f87b7eb66529cf5917499/tests/CodeGen-Binding/Xamarin.Android.JcwGen-Tests/Xamarin.Android.JcwGen-Tests.projitems#L18

But maybe `run-as` is needed for something else?